db/seeds.rb

```diff
@@ -248,6 +248,86 @@
     fund_pledge: 1000)
 end
 
+# create patients with ALL THE INFO to play with archiving
```

Comment:
@colinxfleming @lwaldsc to the best of my knowledge, this is what "complete" patients look like (every option toggled one way or the other). Could you both take a gander? Happy to discuss in person on Thursday if that is easier!